### ISSUE TITLE
There's a mistake in the event loop code in the mdbook

### ISSUE THREAD

**Issue by NoamDev** [glium/glium#1979]
https://github.com/glium/glium/blob/60889a20a9e78ef872eb156f17575c428a1e7464/examples/tutorial-10.rs#L57-L76

The wait until is reset before checking the event, so in case there was some event before the resume event (usually), the next frame is postponed by 1/60 seconds again and again.

It is possible this mistake is present in other chapters of the mdbook, I do not know.

**Comment by est31** [comment]
Good point! Further analysis of the issue and a PR are welcome :). Also don't forget keeping the mdbook itself up to date.
